src/v3/src/components/Checkbox/Checkbox.tsx

```diff
@@ -70,8 +70,8 @@ const Checkbox: UISchemaElementComponent<UISchemaElementComponentWithValidationP
         isDisabled={loading || isReadOnly}
         label={label}
         name={name}
-        onBlur={(e: ChangeEvent<HTMLInputElement>) => {
-          handleBlur?.(e?.currentTarget?.checked);
+        onBlur={(e: FocusEvent) => {
+          handleBlur?.((e?.currentTarget as HTMLInputElement)?.checked, e as FocusEvent);
```

Comment:
See `src/v3/src/types/handlers.ts`, which contains our definitions of the events. You shouldn't need casts with `as`.